accounts/views.py

```diff
@@ -25,13 +28,33 @@ def form_valid(self, form):
             login(self.request,user)
             return response
         
+class Login(LoginView):#ログインページ
+    form_class = LoginForm
+    template_name = 'accounts/login.html'
+
+class Logout(LoginRequiredMixin,LogoutView):#ログアウトページ

```

Comment:
```suggestion
class Logout(LoginRequiredMixin, LogoutView):
```
このように、引数同士も`,`の後はスペース空けてください。